### ISSUE TITLE
Add mechanism for showing when new minor release is available

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#33729]
Suppose you are on 46.7, and 47.0 and 46.8 are both available, we should have some way to let you know that you can upgrade to 46.8 if you're not in the mood for doing a major version upgrade.

Depends on #33728

I believe https://static.metabase.com/version-info.json is just passed thru to the FE as the `version` setting so I think this would only require FE changes.

**Comment by tsmacdonald** [comment]
To clarify the requirement (via @brunobergher ), we should _only_ let the user know about one new version. If the user is on 46.7, we should tell them about 47 and not mention 46.8.

And, of course, if the user is on 47.0 and then 47.1 gets released, we should tell them about that too. This case is the real bug that we're seeing right now.(

**Comment by likeshumidity** [comment]
I think we should tell them about both versions... Right now if someone was on 46.7, I would want them to know about 46.8 and 47.2 (i.e. latest minor/patch if one exists and latest major if a newer one exists).

**Comment by likeshumidity** [comment]
Sorry submitted to soon. My reasoning is:
- new major versions are often unstable until patch .2, .3, or .4, I would not recommend upgrading production to a .0 or .1 (this may change in the future, but I am basing this recommendation on the past few years of release history). Upgrading to a major version may reduce stability.
- new minor and patch versions may include security fixes or bugfixes that will generally improve stability

In order for customers to have the best experience with Metabase, we should strongly encourage them to make upgrades that improve stability (minor and patch upgrades) and gently encourage them to make upgrades that may reduce stability (major upgrades with a minor version less than 3 historically speaking).

**Comment by likeshumidity** [comment]
@brunobergher looping you in re: my last comment to get Product input

**Comment by brunobergher** [comment]
Fair point, showing both in the UI surface we have for it makes sense and doesn't take up too much space.

**Comment by tsmacdonald** [comment]
@brunobergher if this is an official request for showing both we should loop in @metabase/design for what that looks like. CC @npretto .

**Comment by dragonsahead** [comment]
@iethree is this still a thing?

**Comment by iethree** [comment]
> [@iethree](https://github.com/iethree) is this still a thing?

It's still possible if we decide we want to do it. would need a design. Doesn't seem like a priority. I think we will be revisiting this admin section in light of LTS versions soon.
